org.jacoco.tests/pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>../org.jacoco.ant.test</module>
     <module>../jacoco-maven-plugin.test</module>
     <module>../org.jacoco.examples.test</module>
+    <module>../org.jacoco.tests.coverage</module>
```

Comment:
@Godin There are some differences in confuguration e.g. regarding exec file locations which would be an incompatible change of the existing goal. Also this would mean that we create a report group in any case. To move on I would go with the new goal and maybe consolidate later as we learn more about actual usage.